config/orchestrations/sso/0.1/sso.yaml

```diff
@@ -287,7 +287,7 @@ spec:
       - sso-postgresql
       from:
         kind: ImageStreamTag
-        name: postgresql:10
+        name: postgresql:9.6
```

Comment:
I'm asking @brutif if this change was intentional as it's not mentioned in the PR.

Reply:
yes